nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs

```diff
@@ -375,7 +375,7 @@ private static ProjectRootElement CreateProjectRootElement(ProjectBuildFile buil
         {
             var tempProjectPath = await CreateTempProjectAsync(tempDirectory, repoRoot, projectPath, targetFramework, packages);
 
-            var (exitCode, stdout, stderr) = await ProcessEx.RunAsync("dotnet", $"build \"{tempProjectPath}\" /t:_ReportDependencies");
+            var (exitCode, stdout, stderr) = await ProcessEx.RunAsync("dotnet", $"build \"{tempProjectPath}\" /t:_ReportDependencies /p:EnableWindowsTargeting=true");
```

Comment:
Could we move this property into the Directory.Build.props that we write out in the `CreateTempProjectAsync` method? That method is used elsewhere and I want to be sure everyone gets the benefit.

Reply:
Okay, what should I do? Should I change line 363?

Reply:
That's right. You could do something like the following.

```c#
        await File.WriteAllTextAsync(
            Path.Combine(tempDir.FullName,
            "Directory.Build.props"),
            """
            <Project>
              <PropertyGroup>
                <EnableWindowsTargeting>true</EnableWindowsTargeting>
              </PropertyGroup>
            </Project>
            """);
```